combat-trainer.lic

```diff
@@ -3229,6 +3232,7 @@ class AttackProcess
     Flags.add('ct-aim-failed', 'you stop aiming', 'stop concentrating on aiming')
     Flags.add('ct-ranged-ready', 'You think you have your best shot possible now.')
     Flags.add('ct-face-what', 'Face what')
+    Flags.add('war-stomp-ready', 'You feel ready to perform another War Stomp'); Flags['war-stomp-ready'] = true
```

Comment:
```suggestion
    Flags.add('war-stomp-ready', 'You feel ready to perform another War Stomp')
```

Reply:
Setting the flag to true is what Flags.add does.

Reply:
Flags.add sets it to false. It's not true until it matches, at which point checking it returns the matched phrase.